### PR TITLE
Set datasource for Grafana targets

### PR DESCRIPTION
**Pull request by jupblb**
**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
This PR fixes issues with editing Grafana targets when reviewing dashboards.
**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
/assign @mborsz
**Does this PR introduce a user-facing change?**:

```release-note
N/A
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
N/A
```

### CODE REVIEW

**Comment by jupblb** [comment]
Fixed. The PR is now split into:
- https://github.com/kubernetes/perf-tests/pull/1909/commits/37e41cdef985173011b9478ff15f67dcf6338fe6 - Simply regenerating the JSONs based on existing code
- https://github.com/kubernetes/perf-tests/pull/1909/commits/0ebcda06737ea5fd0ae911c7cc5f19a736b59c56 - Made the change of attaching `datasource = "$source"` to all grafana `Target`s and regenerated the JSONs again

File: clusterloader2/pkg/prometheus/manifests/dashboards/dns.dashboard.py [hunk 1]
```diff
@@ -44,10 +44,12 @@
         title="probe: lookup rate",
         targets=[
             g.Target(
+                datasource = "$source",
```
Comment (jkaniuk) [inline clusterloader2/pkg/prometheus/manifests/dashboards/dns.dashboard.py:47]: Why do we need to set it separately?

File: clusterloader2/pkg/prometheus/manifests/dashboards/defaults.py [hunk 2]
```diff
@@ -62,7 +63,11 @@ def simple_graph(title, exprs, legend="", interval="5s", **kwargs):
         # One graph per row.
         targets=[
             g.Target(
-                expr=expr, legendFormat=legend, interval=interval, intervalFactor=1
+                datasource = "$source",
```
[the remark this answers is not in the document]
Comment (jupblb) [reply clusterloader2/pkg/prometheus/manifests/dashboards/defaults.py:66]: Unfortunately not. :(

File: clusterloader2/pkg/prometheus/manifests/dashboards/dns.dashboard.py (hunk 1 above)
Comment (jkaniuk) [reply clusterloader2/pkg/prometheus/manifests/dashboards/dns.dashboard.py:47]: d.Target could handle source, d.TargetWithIntervalSet (naming is hard) could handle interval

**Pull request merged**